data/STRchive-database.json

```diff
@@ -3855,10 +3705,98 @@
         "tr_gnomad":[
             "TR51895"
         ],
-        "disease_description":"Cortical tremor, seizures with generalised motor (tonic-clonic) onset (PMID: 38876750); Adult-onset cortical tremor with epilepsy (PMID: 39349043)",
+        "disease_description":"Cortical tremor, seizures with generalised motor (tonic-clonic) onset [@pmid:38876750]; Adult-onset cortical tremor with epilepsy [@pmid:39349043].",
         "locus_tags":[],
         "disease_tags":[]
     },
+    {
+
+   "chrom":"chr15",
+       "start_hg38":156591766,
+       "stop_hg38":156591783,
+       "start_hg19":156561558,
+       "stop_hg19":156561575,
+       "start_t2t":155728132,
+       "stop_t2t":155728149,
+       "id":"NME_NAXE",
+       "disease_id":"NME",
+       "gene_strand":"+",
+       "reference_motif_reference_orientation":[
+           "GGGCC"
+       ],
+       "pathogenic_motif_reference_orientation":[
+           "GGGCC"
+       ],
+       "pathogenic_motif_gene_orientation":[
+           "GGGCC"
+       ],
+       "benign_motif_reference_orientation":[],
+       "benign_motif_gene_orientation":[],
+       "unknown_motif_reference_orientation":[],
+       "unknown_motif_gene_orientation":[],
+       "disease":"NAXE-related mitochondrial encephalopathy",
+       "gene":"NAXE",
+       "flank_motif":null,
+       "locus_structure":"(GGGCC)*",
+       "inheritance":[
+           "AR"
+       ],
+       "type":"5' UTR",
+       "location_in_gene":null,
+       "benign_min":2.0,
+       "benign_max":7.0,
+       "intermediate_min":null,
+       "intermediate_max":null,
+       "pathogenic_min":200.0,
+       "pathogenic_max":200.0,
+       "ref_copies":null,
+       "motif_len":5,
+       "age_onset":"Predominately infantile onset, extending to 20y or older for general disease; proband onset at one year [@doi:10.1038/s41525-024-00429-5].",
+       "age_onset_min":1,
+       "age_onset_max":1,
+       "typ_age_onset_min":0,
+       "typ_age_onset_max":0,
+       "novel":"ref",
+       "mechanism":"LoF",
+       "mechanism_detail":"Reduced NAXE expression from expansion in promoter [@doi:10.1038/s41525-024-00429-5]",
+       "source":[],
+       "details":"Normal range (2-7) alleles established by 484 control alleles and validated with orthogonal databases, while proband had expansion of ~200 repeats [@doi:10.1038/s41525-024-00429-5].",
```

Comment:
Suggest adding something like: locus is supported by a single individual with two expanded alleles inherited from their mother via uniparental disomy. While the repeat expansion is newly reported, other variants in the NAXE gene have previously been associated with mitochondrial encephalopathy.

Reply:
Also normal -> benign

Reply:
Suggest adding something about this here and/or under mechanism:
"bioinformatics analysis of the kinetics features of PacBio long-read sequencing data detected CpG hypermethylation at the repeat sequence (GGGCC) per se and in the region downstream of the repeat sequence (Fig. [4c, d](https://www.nature.com/articles/s41525-024-00429-5#Fig4)) around the promoter region of the patient’s genome, compared to those in the controls. The long-read sequencing in the mother also showed hypermethylation in the allele with repeat expansion, whereas the allele with a normal range of GGGCC repeat did not show hypermethylation in the repeat and its downstream region (Fig. [4d](https://www.nature.com/articles/s41525-024-00429-5#Fig4))."